storage/cloud-client/encryption_test.py

```diff
@@ -16,13 +16,17 @@
 import tempfile
 
 from google.cloud import storage
+from google.cloud.storage import Blob
 import pytest
 
 import encryption
 
-TEST_ENCRYPTION_KEY = 'brtJUWneL92g5q0N2gyDSnlPSYAiIVZ/cWgjyZNeMy0='
+TEST_ENCRYPTION_KEY = '0YSyzDHlx5QAqSDOnrpqktm3+RkqiP1udwjrk3d7SGk='
```

Comment:
for some reason i had thought the old key wasn't working. seems to work now. changed it back!